src/scenarios/mvcapptemplate/pre.py

```diff
@@ -10,7 +10,7 @@
 setup_loggers(True)
 precommands = PreCommands()
 precommands.new(template='mvc',
-                output_dir=const.APPDIR,
+                output_dir=const.SRCDIR,
```

Comment:
Why? I think the intent is that apps always run from "app" and if there is existing source code it should be checked into "src". If you use the `existing` path from `precommands.py` it copies the `src` to `app` so that build and other verbs work out. Is there a flaw with this scheme?

Reply:
I think this is doing what you said if what ```dotnet new``` generates could be considered as "source code" for SDK scenarios? SDK iteration setup always copies things from ```src``` to ```app``` and do ```dotnet build``` in ```app```. So we need the template goes to ```src``` instead of ```app```.

Reply:
Ah - I see the confusion. My intent is that _static_ source code, that's checked in, goes under `src`. Anything dynamic can go in `app`. The automatic copy from whatever you say the project dir is to `app` happens only if you call [`precommands.existing`.](https://github.com/dotnet/performance/blob/master/src/scenarios/shared/precommands.py#L97)

Reply:
yes wherever the src code is from we just can't have ```dotnet new``` and ```dotnet build``` in the same folder.

Reply:
Also it seems the backup command in ```precommands.py``` is not necessary any more since it's basically ```precommands.existing```...

Reply:
one other point in "generate to `const.APPDIR`"'s favor: we specifically **don't** exclude `const.SRCDIR` from the `.gitignore`, so if there are mistaken changes while someone is working they're discovered.

Reply:
Good to know! This won't affect our case though because ```src``` is not in ```.gitignore``` anyway. The extra overhead is manually not checking in the ```src``` folder on commit for dynamic template tests.

Reply:
I still think this should be `APPDIR`. (As with all the other places you're calling `.new`. Let's chat Monday.